stack.yaml

```diff
@@ -28,8 +28,13 @@ extra-deps:
 - constrained-dynamic-0.1.0.0
 - haddock-api-2.20.0
 - haddock-library-1.6.0
-- haskell-lsp-0.2.2.0
-- haskell-lsp-types-0.2.2.0
+# - haskell-lsp-0.2.2.0
+# - haskell-lsp-types-0.2.2.0
+- github: alanz/haskell-lsp
+  commit: 82a5128e66ea8e4a82fa96826eca2d94f729d8c5
+  subdirs:
+  - .
+  - haskell-lsp-types
```

Comment:
Interesting, did not realise this syntax was a thing

Reply:
Except we are preferring to use submodules for this, else we have to update every stack file, and the cabal build does not work.